extensions/wikia/ArticleVideo/tests/ArticleVideoContextTest.php

```diff
@@ -45,4 +45,66 @@ public function featuredVideoProvider() {
 			] ], 'Featured video not set when data is correct' ],
 		];
 	}
+
+	/**
+	 * @dataProvider relatedVideoProvider
+	 * @param $expected
+	 * @param $wgEnableArticleRelatedVideo
+	 * @param $wgArticleVideoRelatedVideos
+	 * @param $message
+	 */
+	public function testGetRelatedVideo(
+		$expected,
```

Comment:
does the format fit to our style guide? each argument in separate line looks strange for me